ALEC-227: ALEC docs: update installation section (#101)
OpenNMS/alec: docs/modules/admin/pages/overview.adoc

> = Overview
> :imagesdir: ../assets/images

Reviewer: ```suggestion

:imagesdir: ../assets/images
= Overview
```

> This section describes maintenance tasks and more advanced procedures, including distributed deployment and how to upgrade ALEC.

Reviewer: ```suggestion
This section describes maintenance tasks and advanced procedures, including distributed deployment and how to upgrade ALEC.
```

:imagesdir: ../assets/images
= Overview

> This section describes maintenance tasks and advanced procedures, including distributed deployment and how to upgrade ALEC.

Reviewer: ```suggestion
This section describes maintenance tasks and advanced procedures, including distributed installation instructions and how to upgrade ALEC.
```
